oak_functions/lookup_data_generator/src/main.rs

```diff
@@ -19,20 +19,32 @@ use bytes::BytesMut;
 use oak_functions_abi::proto::Entry;
 use prost::Message;
 use rand::Rng;
+use serde::Serialize;
 use std::{fs::File, io::Write};
 use structopt::StructOpt;
 
 #[derive(StructOpt, Clone, Debug)]
 #[structopt(about = "Oak Functions Lookup Data Generator")]
 pub struct Opt {
-    #[structopt(long, default_value = "20")]
-    key_size_bytes: usize,
-    #[structopt(long, default_value = "1000")]
-    value_size_bytes: usize,
-    #[structopt(long, default_value = "100")]
-    entries: usize,
     #[structopt(long)]
     out_file_path: String,
+    #[structopt(subcommand)]
+    cmd: Command,
+}
+
+#[derive(StructOpt, Clone, Debug)]
+pub enum Command {
+    #[structopt(about = "Generate random key value pairs")]
+    Random {
+        #[structopt(long, default_value = "20")]
+        key_size_bytes: usize,
+        #[structopt(long, default_value = "1000")]
+        value_size_bytes: usize,
+        #[structopt(long, default_value = "100")]
+        entries: usize,
+    },
+    #[structopt(about = "Generate entries for the weather lookup example")]
```

Comment:
Could you also add a comment that it's randomly generated?

Reply:
Done.